<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTableWidget</class>
 <widget class="QWidget" name="EditTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditTableWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="addDeleteRowQGroupBox">
     <property name="title">
      <string>Dodaj/Usuń wiersz</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="addRowQRadioButton">
        <property name="text">
         <string>dodaj wiersz</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="deleteRowQRadioButton">
        <property name="text">
         <string>usuń wiersz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="optionsQGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="rowIdQLabel">
        <property name="text">
         <string>id wiersza</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="columnNameQLabel">
        <property name="text">
         <string>nazwa kolumny</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="valueQLabel">
        <property name="text">
         <string>wartość</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="addQPushButton">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="rowIdQSpinBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="columnNameQComboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="valueQLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="deleteQPushButton">
        <property name="text">
         <string>Usuń</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableQTableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
